University-advisor/Forms/SignupForm.cs

```diff
@@ -1,20 +1,111 @@
+                var universityList = new List<string>();
+                foreach (Object[] row in universityResult)
+                {
+                    foreach (object column in row)
+                    {
+                        universityList.Add(column.ToString());
+                    }
+                }
+                universityBox.DataSource = universityList;
+                statusBox.DataSource = statusList;
+            }
+        }
+
+        private void CreateAccountButton_Click(object sender, EventArgs e)
+        {
+            if (ValidateFields())
+            {
+                var newUser = new UserModel()
+                {
+                    Username = usernameText.Text,
+                    FirstName = firstNameText.Text,
+                    LastName = lastNameText.Text,
+                    Email = emailText.Text,
+                    University = universityBox.SelectedItem.ToString(),
+                    Status = statusBox.SelectedItem.ToString(),
+                    Password = passwordText.Text,
+                };
+                SendUserToDb(newUser);
+            }
+        }
+
+        private void SendUserToDb(UserModel newUser)
+        {
+            string txtSqlQuery = "INSERT INTO userData (username, first_name, last_name, email, university, status, password) VALUES ";
+            txtSqlQuery += "('" + newUser.Username + "', '" + newUser.FirstName + "', '" + newUser.LastName + "', '" + newUser.Email + "', '" + newUser.University + "', '" + newUser.Status + "', '" + newUser.Password + "');";
+            try
+            {
+                if (SqlDriver.Execute(txtSqlQuery))
+                {
+                    MessageBox.Show("New user is successfully created");
+                    Logger.Log("New user is successfully created");
+                    Hide();
+                }
+                else
+                {
+                    MessageBox.Show("User cannot be created");
+                    Logger.Log("User cannot be created");
+                }
+            }
+            catch (Exception e)
+            {
+                Logger.Log(e.Message);
+            }
+        }
+
+        private bool ValidateFields()
+        {
+            if (String.IsNullOrEmpty(usernameText.Text) || String.IsNullOrEmpty(firstNameText.Text) ||
+                String.IsNullOrEmpty(lastNameText.Text) || String.IsNullOrEmpty(emailText.Text) ||
+                String.IsNullOrEmpty(passwordText.Text) || String.IsNullOrEmpty(confirmPasswordText.Text))
+            {
+                MessageBox.Show("Do not leave empty fields");
+                return false;
+            }
+
+            var regex = new Regex(@"^([\w\.\-]+)@([\w\-]+)((\.(\w){2,3})+)$");
+            if (!regex.Match(emailText.Text).Success)
+            {
+                MessageBox.Show("Wrong email format");
+                return false;
+            }
+
+            if (!passwordText.Text.Equals(confirmPasswordText.Text))
```

Comment:
I think it would be a good idea to add a requirement that a password has to be at least 6 characters long so it would be more secure.